### ISSUE TITLE
[0.9.4 staging-2015] Command '/test economics' spawns closed stores

### ISSUE THREAD

**Issue by SlayksWood** [StrangeLoopGames/EcoIssues#22175]
- [x] 1. Have crash with tooltip afte using this command.
Step to reproduce:
- create a new world (English), connect to it.
- use /test economics
- write "Store" in chat, mouseover this text to seee tooltip:
![image](https://user-images.githubusercontent.com/45708377/121488635-eb70ef80-c9db-11eb-9b69-66459582fcfe.png)
- crash. 
```
Server encountered an exception:

<size=60,00%>Exception: NullReferenceException
Message:Object reference not set to an instance of an object.
Source:System.Private.CoreLib

System.NullReferenceException: Object reference not set to an instance of an object.
   at System.Object.GetType()
   at Eco.Shared.Networking.RPCManager.TryCreateInvocation[TArg](INetClient client, Object target, String methodName, IList`1 packedArgs, Func`3 unpackArg, RPCInvocation& invocation)
   at Eco.Core.Controller.ControllerManager.HandleViewRPC(INetClient client, Int32 controllerID, String methodName, BSONObject bson)
Outer Exceptions:
Exception has been thrown by the target of an invocation.</size>
```
[rhfi.txt](https://github.com/StrangeLoopGames/EcoIssues/files/6629529/rhfi.txt)
- This does not always happen. I'd say that it happened 60% with English client and 0/3 with gibberish for me. But a coincidence is possible. If you can't catch crash you can download my world and just try to open this tooltip.
Save files: https://drive.google.com/file/d/1ubpt96RePM-gdJ5Y0UD2N6YSgbBRkW9u/view?usp=sharing

- [ ] 2. 3-4/5 Stores that was spawned with command are closed:
![image](https://user-images.githubusercontent.com/45708377/121489447-c4ff8400-c9dc-11eb-84fe-b61a64139e0c.png)

**Comment by SlayksWood** [comment]
I caught this crash and without command so create a separate issue for it #22233

**Comment by TwixterZ** [comment]
Fix Tested and confirmed as of 9.5 Develop-400

only 1 out of 5 test worlds I have the whole game stop working (chat, tooltip)
